CHANGELOG.md

```diff
@@ -114,7 +114,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where special characters where removed from non label key generation pattern parts [#4767](https://github.com/JabRef/jabref/issues/4767)
 - We fixed an issue where the RIS import would overwite the article date with the value of the acessed date [#4816](https://github.com/JabRef/jabref/issues/4816)
 - We fixed an issue where an NullPointer exception was thrown when a referenced entry in an Open/Libre Office document was no longer present in the library. Now an error message with the reference marker of the missing entry is shown. [#4932](https://github.com/JabRef/jabref/issues/4932)
-
+- We fixed an issue where the exception related to timezone was too big. [#4827](https://github.com/JabRef/jabref/issues/4827)
```

Comment:
maybe: where a database exception related to a missing timezone was too big?
(cause otherwise it sounds a bit confusing

Reply:
sure, done